src/readme.txt

```diff
@@ -5,7 +5,7 @@ License:           GNU General Public License v2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
 Requires PHP:      5.6.20
 Requires at least: 5.0
-Tested up to:      5.9
+Tested up to:      6.1
```

Comment:
```suggestion
Tested up to:      6.3
```

Assuming this can be verified by the time [WordPress 6.3](https://make.wordpress.org/core/6-3/) ships, bumping to 6.3 would make sense. Sorry, this is a moving target, @robinwpdeveloper 😂